source/_includes/common-tasks/update.md

```diff
@@ -3,85 +3,60 @@ Best practice for updating Home Assistant Core:
+          - Go to {% my updates title="**System** > **Updates**" %}.
+            - Select the update notification.
+            - Select the cogwheel {% icon "mdi:cog-outline" %}, then set **Visible** to active.
 
-    1. Open your Home Assistant UI.
-    2. Navigate to the **Settings** panel.
-    3. On the top you will be presented with an update notification.
-       - **Troubleshooting**: If you do not see that notification:
-         - In the top right corner, select the three dots {% icon "mdi:dots-vertical" %} menu and select **Check for updates**.
-         - Go to {% my updates title="**System** > **Updates**" %}.
-           - Select the update notification.
-           - Select the cogwheel {% icon "mdi:cog-outline" %}, then set **Visible** to active.
+  - title: Using the CLI
+    content: |
 
-- title: Using the CLI
-  content: |
+      ```bash
+      ha core update --backup
+      ```
 
-    ```bash
-    ha core update --backup
-    ```
+      _The_ `--backup` _flag here ensures that you have a partial backup of your current setup in case you need to downgrade._
 
-    _The_ `--backup` _flag here ensures that you have a partial backup of your current setup incase you need to downgrade._
-
-{% endtabbed_block %}
+  {% endtabbed_block %}
 
 {% elsif page.installation == "container" %}
 
-{% tabbed_block %}
-
-- title: Docker CLI
-  content: |
-
-    **First start with pulling the new container.**
-
-    ```bash
-    docker pull {{ site.installation.container }}:stable
-    ```
-
-    **[You then need to recreate the container with the new image.](/installation/linux#install-home-assistant-container)**
-
-- title: Docker Compose
-  content: |
+  {% tabbed_block %}
 
-    ```bash
-    docker compose pull homeassistant
-    docker compose up -d
-    ```
+  - title: Docker CLI
+    content: |
 
-{% endtabbed_block %}
+      **First start with pulling the new container.**
 
-{% elsif page.installation == "core" %}
+      ```bash
+      docker pull {{ site.installation.container }}:stable
+      ```
 
-1. Stop the Home Assistant service.
+      **[You then need to recreate the container with the new image.](/installation/linux#install-home-assistant-container)**
 
-2. Switch to the user that is running Home Assistant
+  - title: Docker Compose
+    content: |
 
-    ```bash
-    sudo -u homeassistant -H -s
-    ```
+      ```bash
+      docker compose pull homeassistant
+      docker compose up -d
+      ```
 
-3. Activate the virtual environment that Home Assistant is running in
-
-    ```bash
-    source /srv/homeassistant/bin/activate
-    ```
-
-4. Download and install the new version
-
-    ```bash
-    pip3 install --upgrade homeassistant
-    ```
-
-5. When that is complete start the service again for it to use the new files.
+  {% endtabbed_block %}
 
 {% endif %}
+
+4. Review persistent notifications and logs to see if there are any issues with your configuration that need to be addressed.
```

Comment:
Hmm, persistent notifiations are the things in the sidebar, we don't use those anymore for this (last release we deprecated that new device found one). We generally use repair issues that show up in the settings page, but I am not sure what a good name for that is